client/my-sites/people/people-invite-details/index.jsx

```diff
@@ -85,7 +85,7 @@ export class PeopleInviteDetails extends React.PureComponent {
 					scary={ isPending }
 					onClick={ this.handleDelete }
 				>
-					{ isPending ? translate( 'Revoke Invite' ) : translate( 'Clear Invite' ) }
+					{ isPending ? translate( 'Revoke invite' ) : translate( 'Clear invite' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.